bin/server.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+'use strict';
+const CONFIG = require('../_config');
+const express = require('express');
+
+// Create Express App
+const app = express();
+app.use('/hedwig', express.static(CONFIG.srcComponents));
+app.use('/', express.static(CONFIG.srcComponents));
```

Comment:
not really asking for a change here, more of just a question.  What is the difference between `/hedwig` and `/` ?  It appears that they are both serving up the same path.